src/java.compiler/share/classes/javax/lang/model/SourceVersion.java

```diff
@@ -71,7 +71,9 @@ public enum SourceVersion {
      *      record patterns in preview)
      *  20: no changes (pattern matching for switch in fourth preview,
      *      record patterns in second preview)
-     *  21: tbd
+     *  21: pattern matching for switch and record patterns (string
+     *      templates in preview, unnamed patterns and variables, unnamed
+     *      classes and instance main methods in preview
```

Comment:
```suggestion
     *      classes and instance main methods in preview)
```
missing right parenthesis

Reply:
Corrected; thanks @liach .